Models/Question.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace QuizzingApp341.Models;
+
+[Serializable()]
+public class Question : INotifyPropertyChanged
+{
+    String problem;
+    String correntAnswer;
```

Comment:
Not sure if my other comment disappeared or whatever, but did you mean correctAnswer here..

Reply:
Good catch, I went ahead and fixed the spelling mistake